docs/source/developer_guide/plugins/overview.rst

```diff
@@ -59,19 +55,19 @@ It is important to keep in mind that, even though there is very little that is i
 
 Failure to comply with the points below will probably mean that your plugin will not be able to be listed as an officially endorsed plugin.
 
-A plugin may / should not
+A plugin may/should not:
```

Comment:
I think the meaning you have in mind is best conveyed by:
A plugin cannot (should not):
Ex:
Users may not use the querybuilder if they know how to use SQL.
Users cannot push changes direcctly into aiida_core/develop